auth-web/src/stores/org.ts

```diff
@@ -884,6 +890,12 @@ export const useOrgStore = defineStore('org', () => {
     return response?.data
   }
 
+  async function removeOrgProducts (productCode: string): Promise<OrgProduct> {
```

Comment:
observation: I don't think adding to org store is a good idea at this point, look how big it is

Reply:
moved to user store